docs/docsite/rst/playbook_guide/playbooks_variables.rst

```diff
@@ -112,14 +112,14 @@ While documentation examples focus on ``true/false`` to be compatible with ``ans
 .. table::
    :class: documentation-table
 
-   =============================================================================================== ====================================================================
-    Valid values                                                                                    Description
-   =============================================================================================== ====================================================================
-    ``True`` , ``'true'`` , ``'t'`` , ``'yes'`` , ``'y'`` , ``'on'`` , ``'1'`` , ``1`` , ``1.0``     Truthy values
+   ========================================================================================================== ====================================================================
+    Valid values                                                                                               Description
+   ========================================================================================================== ====================================================================
+    ``True`` , ``'True'``, ``'true'`` , ``'t'`` , ``'yes'`` , ``'y'`` , ``'on'`` , ``'1'`` , ``1`` , ``1.0``     Truthy values
```

Comment:
Why add `'True'` instead of mentioning that the string values are case insensitive? Otherwise we're leaving out things like `'TRUE'`, `'truE'`, etc.